Azure-Samples/cosmos-db-nosql-dotnet-samples
# fix typo: from alredy to already

## Purpose

fix a typo

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
pull code, verify typo fixed

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
manually review the doc
```

## What to Check
Verify that the following are valid
* 'alredy' changed to 'already'

## Other Information
